### PR TITLE
Move alerting issues into a zenhub epic

### PR DESCRIPTION
**Pull request by ChadNedzlek**

### CODE REVIEW

File: src/DotNet.Status.Web/ZenHubClient.cs [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        private readonly IOptionsMonitor<ZenHubOptions> _options;
+        private readonly ILogger<ZenHubClient> _logger;
+
+        public ZenHubClient(IOptionsMonitor<ZenHubOptions> options, ILogger<ZenHubClient> logger)
+        {
+            _options = options;
+            _logger = logger;
+        }
+
+        public async Task AddIssueToEpic(IssueIdentifier child, IssueIdentifier epic)
```
Comment (JohnTortugo) [inline src/DotNet.Status.Web/ZenHubClient.cs]: Async suffix?

File: src/DotNet.Status.Web/ZenHubClient.cs [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+                    remove_issues = Array.Empty<string>(),
+                    add_issues = new[]
+                    {
+                        new {repo_id = child.RepositoryId, issue_number = child.IssueId}
+                    }
+                });
+
+                var content = new StringContent(body, Encoding.UTF8, "application/json");
+
+                _logger.LogInformation("Moving issue {child_repo}#{child_issue} under epic {epic_repo}#{epic_issue}", child.RepositoryId, child.IssueId, epic.RepositoryId, epic.IssueId);
```
Comment (JohnTortugo) [inline src/DotNet.Status.Web/ZenHubClient.cs]: NIT: Can't we use string interpolation here?

Comment (alexperovich) [reply src/DotNet.Status.Web/ZenHubClient.cs]: For this logger this format is preferred over string interpolation. The name in `{}` in the format string is used as a key for the corresponding value. In this case the log message will have a "child_repo" property that is child.RepositoryId. The same will happen to all the others.

Comment (ChadNedzlek) [reply src/DotNet.Status.Web/ZenHubClient.cs]: No. The logger is magic.  The placeholders create variables in the message that can be independently queried... you can make a query that says "show me all the messages where child_repo == 'Pizza'".

It's... weird magic, but magic it is. :-)

**Pull request merged**